src/components/RecipesList.js

```diff
@@ -36,21 +43,31 @@ class RecipesList extends React.Component {
         let recipes = data.map((recipe) => {
           return {id:recipe.id, title:recipe.title, image:recipe.image};
         });
+        this.setState({recipes: recipes})
+        this.setState({ingredients: []})
         this.setState({recipes: recipes, showDataLoading: false})
```

Comment:
Can we combine all of these setStates into one? Like the last setState you should be able to just add that in as well.

Reply:
Yes